__tests__/app.test.js

```diff
@@ -29,3 +29,29 @@ describe("GET/api/topics", () => {
       });
   });
 });
+
+describe("GET/api", () => {
+  it("provide a description of all other endpoints available", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        const keys = Object.keys(body);
```

Comment:
I would still like this to be returned on a key of endpoints. You could also consider reading the file into the tests and just checking the file contents get returned from the endpoint